src/background/index.ts

```diff
@@ -57,10 +71,14 @@ chrome.webRequest.onBeforeRequest.addListener(handleBeforeRequest, { urls: ['<al
     'blocking',
 ]);
 
+const extraInfos = ['requestHeaders', 'blocking'];
+if (getBrowser() === 'Chrome') {
```

Comment:
maybe we can have a const dicvtionary with browser
something like
```typescript
const BROWSERS = {
  CHROME: 'Chrome',
  FIREFOX: 'Firefox',
  EDGE: 'Edge',
} as const;
type BROWSERS = typeof BROWSERS[keyof typeof BROWSERS];
```

Reply:
I changed to enum instead then.

Reply:
`enum` is not well built in typescript, and can rapidly makes code hard to read. Using a const map works better.